pkg/templates/externalccm/vsphere.go

```diff
@@ -0,0 +1,326 @@
+/*
+Copyright 2019 The KubeOne Authors.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
```

Comment:
```suggestion
You may obtain a copy of the License at

    http://www.apache.org/licenses/LICENSE-2.0

Unless required by applicable law or agreed to in writing, software
```

Seems like you've missed couple of \n, so now boilerplate verifier is complaining.